### DISCUSSION TITLE
Session Time / Expiry configuration?

### DISCUSSION THREAD

**Discussion by prologic** [outline/outline#5760] [Self Hosting] [answered]
Hi Tema 👋

I can't seem to find a configuration item for the amount of time a logged in session lasts for. I'm using OIDC for authentication (_no other method_) and the session seems to only last a few hours. Is there a environment variable to control this? What's the default? I can't find any docs on this.

Kind regards

James

**Comment by tommoor** [comment, marked as answer]
Outline will stay logged in for as long as the OIDC session is valid.  Make sure that your OIDC provider is issuing a refresh token with a long lifespan to enable the app to stay logged in, as it will check the session validity on each load

**Reply by prologic** [reply]
Ahh! This is an OIDC thing, thanks! 😅
